src/utils/pages.ts

```diff
@@ -7,4 +7,10 @@ export enum Pages {
   smSystemTransfers = '/sm-system/transfers',
   smSystemImportProducts = '/sm-system/import-products',
   smSystemProductsDocuments = '/sm-system/products-documents',
+  smSystemSuppliersOrders = '/sm-system/suppliers-orders',
+  smSystemSuppliers = '/sm-system/suppliers-orders/suppliers',
+  smSystemSuppliersDetails = '/sm-system/suppliers-orders/suppliers/:supplierId',
+  smSystemProducts = '/sm-system/suppliers-orders/products',
+  smSystemOrders = '/sm-system/suppliers-orders/orders',
+  smSystemOrdersDetails = '/sm-system/suppliers-orders/orders/:orderId',
```

Comment:
```suggestion
  smSystemOrderDetails = '/sm-system/suppliers-orders/orders/:orderId',
```
po prostu bez liczby mnogiej :D